freva-rest/src/freva_rest/databrowser_api/endpoints.py

```diff
@@ -83,20 +57,24 @@ async def overview() -> SearchFlavours:
     for climate datasets, and each standard offers specific attributes for
     searching and filtering datasets.
     """
+    user_name = current_user.preferred_username if current_user else "global"
+    solr_instance = Solr(server_config)
+
+    all_flavour_responses = await solr_instance.get_all_flavours(user_name)
+    flavours = [f.flavour_name for f in all_flavour_responses]
     attributes = {}
-    for flavour in Translator.flavours:
-        translator = Translator(flavour)
-        if flavour in ("cordex",):
-            attributes[flavour] = list(translator.forward_lookup.values())
+
+    for flavour_response in all_flavour_responses:
+        flavour_name = flavour_response.flavour_name
+        if flavour_name in ("cordex",):
+            attributes[flavour_name] = list(flavour_response.mapping.values())
         else:
-            attributes[flavour] = [
-                f
-                for f in translator.forward_lookup.values()
-                if f not in translator.cordex_keys
+            cordex_keys = ("rcm_name", "driving_model", "rcm_version")
```

Comment:
Don't do this. 

Don't start with if something is something very specific then to this.

Abstract, and find a solution with code that's not checking for specific edge cases.